src/Svg.Model/SvgExtensions.Painting.cs

```diff
@@ -1004,7 +1004,8 @@ internal static bool IsValidStroke(SvgElement svgElement, SKRect skBounds)
         };
 
         var server = svgVisualElement.Fill;
-        var opacity = AdjustSvgOpacity(svgVisualElement.FillOpacity);
+        var opacity = AdjustSvgOpacity(svgVisualElement.Opacity);
+        opacity *= AdjustSvgOpacity(svgVisualElement.FillOpacity);
```

Comment:
This breaks stuff

![image](https://user-images.githubusercontent.com/2297442/186350689-5e6ae8af-bd01-4965-97dc-6c7035a94940.png)